packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -101,16 +97,6 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     this._labelId = getId('Label');
     this._inputId = getId('input');
     this._spinningByMouse = false;
-
-    if (!props.defaultValue && props.value !== undefined) {
-      this._onValidate = props.onValidate;
-      this._onIncrement = props.onIncrement;
```

Comment:
Possible to add a test to validate this is fixed?

Reply:
Added some tests